keyboards/basekeys/slice/rev1_rgb/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+TAP_DANCE_ENABLE = no
+VIA_ENABLE = yes
+BACKLIGHT_ENABLE = no
+SPACE_CADET_ENABLE = no
```

Comment:
```suggestion
VIA_ENABLE = yes
```
I don't see these enabled in the keyboard-level `rules.mk`.

Reply:
I committed these changes.